functions.json

```diff
@@ -2,81 +2,57 @@
     {
         "title": "NodeInfo",
         "description": "Get info about the machine that you're deployed on. Tells CPU count, hostname, OS, and Uptime",
-        "image": "functions/nodeinfo:arm64",
+        "image": {
+            "arm64": "functions/nodeinfo:arm64",
+            "armhf": "functions/nodeinfo:latest-armhf",
+            "x86_64": "functions/nodeinfo:latest"
+        },
         "name": "nodeinfo",
         "fprocess": "node main.js",
         "network": "func_functions",
-        "repo_url": "https://github.com/openfaas/faas/tree/master/sample-functions/NodeInfo",
-        "platform": "arm64"
+        "repo_url": "https://github.com/openfaas/faas/tree/master/sample-functions/NodeInfo"
     },
     {
         "title": "sha512sum",
         "description": "Generate shasums in 512 format",
-        "image": "functions/alpine:latest-arm64",
+        "image": {
+            "arm64": "functions/alpine:latest-arm64"
+        },
         "name": "sha512sum",
         "fprocess": "sha512sum",
         "network": "func_functions",
-        "repo_url": "https://github.com/openfaas/faas/tree/master/sample-functions/AlpineFunction",
-        "platform": "arm64"
-    },
-    {
-        "title": "NodeInfo",
-        "description": "Get info about the machine that you're deployed on. Tells CPU count, hostname, OS, and Uptime",
-        "image": "functions/nodeinfo:latest-armhf",
-        "name": "nodeinfo",
-        "fprocess": "node main.js",
-        "repo_url": "https://github.com/openfaas/faas/tree/master/sample-functions/NodeInfo",
-        "platform": "armhf"
+        "repo_url": "https://github.com/openfaas/faas/tree/master/sample-functions/AlpineFunction"
     },
     {
         "title": "Figlet",
         "description": "Generate ASCII logos with the figlet CLI",
-        "image": "functions/figlet:latest-armhf",
+        "image": {
+            "armhf": "functions/figlet:latest-armhf",
+            "x86_64": "functions/figlet:0.9.6"
+        },
         "name": "figlet",
         "fprocess": "figlet",
-        "repo_url": "https://github.com/openfaas/faas/tree/master/sample-functions/figlet",
-        "platform": "armhf"
-    },
-    {
-        "title": "Left-Pad",
-        "description": "left-pad on OpenFaaS",
-        "image": "rgee0/faas-leftpad",
-        "name": "leftpad",
-        "repo_url": "https://github.com/faas-and-furious/faas-leftpad",
-        "platform": "armhf"
-    },
-    {
-        "title": "Manifest Query",
```

Comment:
Not sure why this is in [store-armhf.json](https://github.com/openfaas/store/blob/9e492dfd956adaf499449315542b996c2a452f21/store-armhf.json#L26) and [store.json](https://github.com/openfaas/store/blob/9e492dfd956adaf499449315542b996c2a452f21/store.json#L185) when the images are the same `rgee0/of-mquery:1.2` so I presumed `x86_64` architecture

Reply:
It's a multi-arch image - works on both

Reply:
Ok thanks for the input @rgee0 I think this is ready.